setup/install_gatling/30_post_install/70_disable_firewall.rb

```diff
@@ -0,0 +1,4 @@
+step "disable firewall" do
+  on(master, "puppet resource service firewalld ensure=stopped")
+  on(master, "puppet resource service firewalld ensure=disabled")
```

Comment:
Beaker **should** have built-in functionality to manage firewalls generically. This is very el7 specific code, which is a pain when dealing with Amazon Linux, which is el6 based.